ngx-fudis/projects/ngx-fudis/src/lib/components/section/section.component.html

```diff
@@ -3,7 +3,7 @@
   fudisGrid
   [id]="id"
   [tabIndex]="-1"
-  [attr.aria-describedby]="_headingId"
+  [attr.aria-labelledby]="_headingId"
```

Comment:
This fixed the landmark issue in Section Component but quick test with Expandable Component showed that our expandables are not listed in screen reader's (VoiceOver) landmark list. So probably we should change the `aria-describedby` to `aria-labelledby` there since it is using native `section` element as well.